test/ffmpeg-qsv/encode/hevc.py

```diff
@@ -175,7 +175,7 @@ def test_r2r(self, case, gop, slices, bitrate, fps, quality, refs, profile):
 
   # TCBRC is VBR LP + LDB + Strict(-1)
   @slash.parametrize(*gen_hevc_tcbrc_parameters(spec, ['main']))
-  def test_tcbrc(self, case, bitrate, fps, profile):
```

Comment:
Changing an existing test function signature/params is risky and generally not acceptable.  It can break other component tests if not carefully done correctly.  It also messes up CI test run histories.

It would be better to create a new test function and parameter generator to handle seek cases specifically, e.g. `test_tcbrc_seek`.

Reply:
Also, is it necessary to test seek with all the different encode combinations (e.g. vbr, cbr, tcbrc, etc..)?  Maybe we only need a simple seek test that uses middleware defaults?